src-ghc/Pact/Interpreter.hs

```diff
@@ -187,7 +187,7 @@ setupEvalEnv
   -> ExecutionConfig
   -> IO (EvalEnv e)
 setupEvalEnv dbEnv ent mode msgData refStore gasEnv np spv pd ec = do
-  gasRef <- newIORef 0
+  gasRef <- newIORef mempty
```

Comment:
I don't like using `mempty` in a non-abstract context, it adds no generality but takes away from what might be understood by specificity.